generators/python-v2/sdk/src/SdkGeneratorCli.ts

```diff
@@ -46,6 +47,16 @@ export class SdkGeneratorCli extends AbstractPythonGeneratorCli<SdkCustomConfigS
     }
 
     protected async generate(context: SdkGeneratorContext): Promise<void> {
+        if (context.customConfig.generateWireTests === true) {
+            try {
+                const wireTestGenerator = new WireTestGenerator(context);
+                await wireTestGenerator.generate();
+                context.logger.info("Generated wire tests successfully");
+            } catch (error) {
+                context.logger.warn(`Failed to generate wire tests: ${error}`);
```

Comment:
let's throw here (meaning no wrap in try-catch).

After all, we have a config flag for this, so users can disable it if they have to. But would like to avoid silent failures